cognite_toolkit/_cdf_tk/commands/modules.py

```diff
@@ -280,7 +307,9 @@ def init(
             organization_dir = Path(organization_dir_raw.strip())
 
         modules_root_dir = organization_dir / MODULES
-        packages = Packages().load(self._builtin_modules_path)
+
+        cdf_toml = CDFToml.load()
+        packages = self._get_library_packages(cdf_toml) or Packages.load(self._builtin_modules_path)
```

Comment:
I think we need an alpha flag here. We should not change the current behavior unless it is set.